moveit_kinematics/kdl_kinematics_plugin/src/kdl_kinematics_plugin.cpp

```diff
@@ -268,7 +268,9 @@ bool KDLKinematicsPlugin::initialize(const rclcpp::Node::SharedPtr& node, const
 
 bool KDLKinematicsPlugin::timedOut(const rclcpp::Time& start_time, double duration) const
 {
-  return ((node_->now() - start_time).seconds() >= duration);
+  static rclcpp::Clock clock{ RCL_STEADY_TIME };
```

Comment:
Can you make `clock` a class member `steady_clock_`?

Reply:
`main` only builds for galactic or rolling, I presume that is why it does not build for you.  If it is something else please let us know and we are happy to help.  Again, thank you for this change.